presto-main/src/main/java/com/facebook/presto/operator/aggregation/noisyaggregation/NoisySumAvgAggregationUtils.java

```diff
@@ -76,24 +76,43 @@ public static void combineNoisySumStates(NoisySumState state, NoisySumState othe
         state.setSum(state.getSum() + otherState.getSum());
     }
 
-    public static void writeNoisySumStateOutput(NoisySumState state, BlockBuilder out)
+    public static void writeNoisySumStateOutput(NoisySumAvgState state, BlockBuilder out)
```

Comment:
nit: I think I'd be inclined to drop the word `State` from the function name. We're writing a noisy sum to output here. The state is passed as input (which is evident from the method signature).